script.js

```diff
@@ -0,0 +1,156 @@
+// namespace object
+const app = {};
+
+// Define Variables:
+app.apiKey = 'c8f1c1da6fe84ef6b510afbd3ad28f27';
+// app.apiKey = "20a367ef2c2e4d4380d95b890faae49b";
+app.apiUrl = "https://api.spoonacular.com/recipes/complexSearch"
+
+// global array of cuisines already excluded from user choices
+const excludedCuisines = [
+    'African',
+    'British',
+    'Cajun',
+    'Caribbean',
+    'Eastern European',
+    'European',
+    'German',
+    'Irish',
+    'Jewish',
+    'Latin American',
+    'Mediterranean',
+    'Nordic',
+    'Southern',
+    'Vietnamese'
+];
+
+// function to display cuisine options in HTML
+app.displayCuisines = () => {
+    // array of cuisine options 
+    const cuisineOptions = ['American', 'Chinese', 'Italian', 'Mexican', 'Thai', 'Japanese', 'French', 'Indian', 'Greek', 'Spanish', 'Korean', 'Middle Eastern'];
+
+    // create array of random cuisines
+    const randomCuisines = [];
+    while (randomCuisines.length < 4) {
+        const randomIndex = Math.floor(Math.random() * cuisineOptions.length);
+        const randomCuisine = cuisineOptions[randomIndex];
+        if (!randomCuisines.includes(randomCuisine)) {
+            randomCuisines.push(randomCuisine);
+        }
+    }
+
+    // loop over each cuisine item in randomCuisines array to create HTML elements and append to form
+    randomCuisines.forEach((cuisine) => {
+        // target HTML to append checkboxes
+        const cuisineChoices = document.querySelector('.cuisineChoices');
+
+        // create div element to populate with label and input
+        const optionButton = document.createElement('div');
+        optionButton.classList.add('optionButton');
+
+        // add input/label HTML to div
+        optionButton.innerHTML =
+            `<input type="checkbox" id="cuisineOption" name="cuisineOption" value="${cuisine}">
```

Comment:
would need to specify which label links to which input

Reply:
added index as parameter to .forEach and included it in the innerHTML to distinguish input/labels more clearly (ex. id="cuisineOptions${index}").